ruby/red-arrow/lib/arrow/table.rb

```diff
@@ -197,10 +197,52 @@ def each_record_batch
     alias_method :length, :n_rows
 
     alias_method :[], :find_column
+    alias_method :slice_raw, :slice
```

Comment:
Could you put a new line between `alias_method :[]` and `alias_method :slice_raw`?
Because they aren't related.

Reply:
I've fixed it.